src/tests/ftest/slurm_setup.py

```diff
@@ -162,6 +158,10 @@ def start_munge(args):
         args (Namespace): Commandline arguments
 
     """
+    if not args.sudo:
+        sudo = ""
+    else:
+        sudo = "sudo"
```

Comment:
Just FYI, this is different in `update_config_cmdlist` vs `start_munge` (but same result)

Reply:
thanks missed it!